packages/taquito-contracts-library/src/errors.ts

```diff
@@ -1,12 +1,4 @@
-/**
- *  @category Error
- *  @description Error that indicates an invalid address being used or passed
- */
-export class InvalidAddressError extends Error {
-  constructor(message: string) {
-    super(message);
-  }
-}
+export * from '@taquito/core';
```

Comment:
Would like to understand is there a reason behind exporting the entire @taquito/core package here instead of specific classes used in this package like the rest of the code change?

Reply:
this is to just stop breaking changes from happening for moving errors from thats that they would otherwise have been coming from

Reply:
in case anyone is importing from the error file directly